config.rb

```diff
@@ -2,6 +2,8 @@
 require 'compass/import-once/activate'
 # Require any additional compass plugins here.
 
+# use UTF-8
+Encoding.default_external = "utf-8"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols.